docs/MAVLINKX.md

```diff
@@ -0,0 +1,15 @@
+# mLRS Documentation: MavlinkX #
+
+([back to main page](../README.md))
+
+MavlinkX is a technology which weeds out deficiencies of the original MAVLink protocol and makes the over-the-air MAVLink communication significantly more robust, reducing packet losses to the minimum. Furthermore, it can provide per-message compression of the MAVLink data stream.
+
+A major deficiancy of the MAVLink protocol is that, in the presence of data loss, it is prone to miss MAVLink packets which are actually valid, i.e., which are correctly and fully recieved and thus provide valid data, but are missed in the parsing of the stream (for a deeper discussion of such topics see [here](https://github.com/mavlink/mavlink/issues/1347)).
+
+Loosing valid packets for "no reason" is obviously highly undesirable, especially given that the bandwidth of long-range links is generally constraint and should be used in the best possible manner. MavlinkX overcomes this issue by reframing the MAVLink messages and using parsing strategies which enables the parser to recover any valid packet from the lossy data stream.
+
+Moreover, MavlinkX provides compression of the MAVLink data stream. Given the nature of the MAVLink data, the average compression rate cannot be huge (simple theoretical best-case estimates suggest ca 20%), but for low-bandwith links this can be sufficient to be significant.
```

Comment:
Same thought as above, should there be a compression heading?

Slight rework:

'MavlinkX provides compression of the MAVLink data stream. Given the nature of MAVLink data, the average compression rate will range from 10 to 20%, which can be very beneficial for low-bandwidth links.'

Reply:
GREAT. Done, with a tiny alternation.